### PR TITLE
chore: [app-router-migration 17]: migrate payments page

### PR DESCRIPTION
**Pull request by dmytro068**
## What does this PR do?

migrate payments page

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: apps/web/middleware.ts [hunk 1]
```diff
@@ -118,6 +118,7 @@ export const config = {
     "/apps/routing_forms/:path*",
     "/event-types",
     "/future/event-types/",
+<<<<<<< HEAD
```
Comment (keithwillcode) [inline apps/web/middleware.ts:121]: This git data needs to be removed

File: packages/app-store/apps.keys-schemas.generated.ts [hunk 2]
```diff
@@ -55,6 +56,7 @@ export const appKeysSchemas = {
   make: make_zod_ts,
   matomo: matomo_zod_ts,
   metapixel: metapixel_zod_ts,
+  "mock-payment-app": mock_payment_app_zod_ts,
```
Comment (keithwillcode) [inline packages/app-store/apps.keys-schemas.generated.ts:59]: I don't think we should have this in the actual list of apps in the app store. We need to put this in a place that's specific to the E2E tests

**Review by keithwillcode — CHANGES_REQUESTED** [review]

File: apps/web/middleware.ts (hunk 1 above)
Comment (hbjORbj) [reply apps/web/middleware.ts:121]: Addressed

File: packages/app-store/apps.keys-schemas.generated.ts (hunk 2 above)
Comment (hbjORbj) [reply packages/app-store/apps.keys-schemas.generated.ts]: Addressed

**Review by keithwillcode — CHANGES_REQUESTED** [review]
Type checks are now failing

**Review by keithwillcode — APPROVED** [review]

**Pull request merged**